### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by joakim-hove**
These are small improvements / fixes ripped out from another feature branch.

### CODE REVIEW

File: libecl/src/ecl_util.c [hunk 1]
```diff
@@ -234,17 +234,17 @@ char * ecl_util_alloc_base_guess(const char * path) {
...
-    util_abort("%s: can not determine timestep from filename:%s - aborting \n",__func__ , filename);
+    return -1;
 
-  if (ext[1] == 'X' || ext[1] == 'F' || ext[1] == 'S' || ext[1] == 'A')
-    report_nr = atoi(&ext[2]);
-  else
-    util_abort("%s: Filename:%s not recognized - valid extensions: Annnn / Xnnnn / Fnnnn / Snnnn - aborting \n",__func__ , filename);
+  if (ext[1] == 'X' || ext[1] == 'F' || ext[1] == 'S' || ext[1] == 'A') {
+    int report_nr = atoi(&ext[2]);
+    return report_nr;
```
Comment (pgdr) [inline libecl/src/ecl_util.c]: just return `atoi...`

File: libert_util/include/ert/util/set.h [hunk 2]
```diff
@@ -36,6 +36,7 @@ set_type   * set_alloc_empty();
 bool         set_add_key(set_type * , const char * );
 bool         set_has_key(const set_type * set, const char * );
 void         set_free(set_type * );
+void         set_free__(void * );
```
Comment (pgdr) [inline libert_util/include/ert/util/set.h]: should `set_free__` be called from somewhere?

File: libecl/src/ecl_grav_calc.c [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-    
-   ERT is distributed in the hope that it will be useful, but WITHOUT ANY 
-   WARRANTY; without even the implied warranty of MERCHANTABILITY or 
-   FITNESS FOR A PARTICULAR PURPOSE.   
-    
-   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html> 
-   for more details. 
+   Copyright (C) 2011  Statoil ASA, Norway.
+
+   The file 'ecl_grav.c' is part of ERT - Ensemble based Reservoir Tool.
```
Comment (pgdr) [inline libecl/src/ecl_grav_calc.c]: this file ...

File: libecl/include/ert/ecl/ecl_grav_calc.h [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-    
-   ERT is distributed in the hope that it will be useful, but WITHOUT ANY 
-   WARRANTY; without even the implied warranty of MERCHANTABILITY or 
-   FITNESS FOR A PARTICULAR PURPOSE.   
-    
-   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html> 
-   for more details. 
+   Copyright (C) 2011  Statoil ASA, Norway.
+
+   The file 'ecl_grav.h' is part of ERT - Ensemble based Reservoir Tool.
```
Comment (pgdr) [inline libecl/include/ert/ecl/ecl_grav_calc.h]: this file ...

File: libecl/src/ecl_nnc_export.c [hunk 5]
```diff
@@ -140,6 +140,8 @@ bool ecl_nnc_equal( const ecl_nnc_type * nnc1 , const ecl_nnc_type * nnc2) {
 
   if (ecl_nnc_sort_cmp( nnc1 , nnc2) == 0)
     return ((nnc1->trans == nnc2->trans) && (nnc1->input_index == nnc2->input_index));
+  else
+    return false;
```
Comment (pgdr) [inline libecl/src/ecl_nnc_export.c]: Wasn't this already merged in somewhere?  I might be wrong here.

**Comment by joakim-hove** [comment]
OK - minor changes addressed.

**Pull request merged**